### PR TITLE
feat(panel): add panel component

### PR DESCRIPTION
**Pull request by rodrigosakamoto**
## Description
[Driver Profile Panel](https://trello.com/c/0z75uvGt/1147-driver-profile-panel)

## Screenshots (if appropriate):
Screenshot from 2022-09-20 12-15-32

## Types of changes

- [ ] Fix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Build (Changes that affect the build system or dependencies)
- [ ] Chore (Other changes that do not modify src or test files)
- [ ] CI (Changes to our CI configration and scripts)
- [ ] Docs (Changes to documentation files)
- [ ] Style (Changes to coding style, like linters and prettier)
- [ ] Refactor (Changes that do not add new functionality or alter current ones)
- [ ] Performance (Performance improvements)
- [ ] Test (Adding or changing tests or the testing setup)
- [ ] Revert (Reverting a previous change)

## Checklist:

- [ ] I have made sure my changes work properly on iOS devices running Safari.
- [ ] I have made sure my changes work properly on Android devices running Chrome.
- [ ] I have made sure my changes work properly on Desktop.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://111--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://111--logrock-pebbles.netlify.app&speed=1

File: lib/components/Panel/Panel.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  svg {
+    margin-left: 18px;
+
+    color: ${({ theme }) => theme.colors.primary[500]};
+  }
+`;
+
+const Panel: FC<PanelProps> = ({ panelsItems, ...props }) => {
+  const [selected, setSelected] = useState<number | null>(null);
```
Comment (cld-santos) [inline lib/components/Panel/Panel.tsx]: I think `expanded` is a more meaningful name for this property

Comment (cld-santos) [reply lib/components/Panel/Panel.tsx]: why it can be true or false?

Comment (cld-santos) [reply lib/components/Panel/Panel.tsx]: got your point, nevermind

**Review by cld-santos — APPROVED** [review]

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://111--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://111--logrock-pebbles.netlify.app&speed=1

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://111--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://111--logrock-pebbles.netlify.app&speed=1

**Pull request merged**